docs/api/native-image.md

```diff
@@ -147,7 +159,7 @@ console.log(image)
 
 Returns `NativeImage`
 
-Creates a new `NativeImage` instance from `buffer`.
+Creates a new `NativeImage` instance from `buffer`. Tries to decode as PNG first.
```

Comment:
It also tries to decode as JPEG:

https://github.com/electron/electron/blob/master/atom/common/api/atom_api_native_image.cc#L92-L95

Reply:
added